espmonitor/src/main.rs

```diff
@@ -15,57 +15,23 @@
 // You should have received a copy of the GNU General Public License
 // along with ESPMonitor.  If not, see <https://www.gnu.org/licenses/>.
 
-use espmonitor::{AppArgs, Chip, Framework, run};
-use pico_args::Arguments;
-use std::convert::TryFrom;
-use std::error::Error;
+use clap::Parser;
+use espmonitor::{AppArgs, run};
 
 fn main() {
     #[cfg(windows)]
     let _ = crossterm::ansi_support::supports_ansi();
     // supports_ansi() returns what it suggests, and as a side effect enables ANSI support
 
-    match parse_args().and_then(|args| args.map(run).unwrap_or(Ok(()))) {
+    let mut args = AppArgs::parse();
+    // TODO: This feels wrong...
+    args.reset = !args.no_reset;
```

Comment:
I don't love this either, but even looking at clap's builder API (the `Arg` struct), I can't seem to figure out how to do this, so I guess we'll just live with it for now.